tasks/skaffold.py

```diff
@@ -0,0 +1,173 @@
+"""
+Skaffold related tasks
+"""
+
+from invoke import UnexpectedExit, task
+from invoke.exceptions import Exit
+
+from tasks.devcontainer import DEVCONTAINER_IMAGE, DEVCONTAINER_NAME
+from tasks.libs.common.color import Color, color_message
+from tasks.libs.common.utils import is_installed
+
+DATADOG_AGENT_MOUNT = "/home/datadog/go/src/github.com/DataDog/datadog-agent"
+
+
+@task
+def minikube_start(ctx, path: str = ".") -> None:
```

Comment:
Do we need this task and `devcontainer_start` available as well? It seems that the `dev` and `create` take care of everything. Maybe for ergonomics reducing the number of available tasks is better 😄

Reply:
It's probably a bit overkill, I wanted to have the possibility to run those tasks separately if needed. Would you prefer for me to move the code in the `create` task?